src/Plugin.php

```diff
@@ -348,7 +348,7 @@ public static function enqueueGaDataLayerScripts() {
   public static function buildAttributesDataTags(array $attributes) {
     array_walk($attributes, function (&$value, $key) {
       if (is_array($value)) {
-        $value = implode($value, ', ');
+        $value = implode(', ', $value);
```

Comment:
For such lists I'd suggest to avoid the whitespace in between the delimiter to make it easier to work with those lists. Otherwise you might have to trim the whitespace when exploding etc.